src/Common/MetricHelper.cs

```diff
@@ -127,7 +127,7 @@ public MetricHelper(INetworkHelper network)
 #if DEBUG
             if (TestMockSupport.RunningMocked)
             {
-                TelemetryConfiguration.Active.DisableTelemetry = true;
+                telemetryConfiguration.DisableTelemetry = true;
```

Comment:
The active telemetry configuration being used may not be `telemetryConfiguration`, so I suppose `TelemetryConfiguration.Active` makes more sense here

Reply:
Using TelemetryConfiguration.Active is not recommended on .NET Core. See https://github.com/microsoft/ApplicationInsights-dotnet/issues/1152 for more details